app/css/index.scss

```diff
@@ -283,6 +283,14 @@
     float: right;
     font-size: 0.8em;
   }
+  .search-result-path {    
+    font-size: 0.7em;
+    font-style: italic;
+    color: gray;
+    text-decoration-line: underline;
+    font-weight: bold;
```

Comment:
```suggestion
    font-size: 0.9em;
    font-style: italic;
    color: #999;
```

Reply:
Will leave you all with the final decision on styling! :)